src/pages/Thanks/ThanksPage.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<www-page data-testid="thanks-page" :class="{'tw-bg-eco-green-1': showNewTYPage && !isOnlyDonation}">
+	<www-page data-testid="thanks-page" :class="{'tw-bg-eco-green-1 auto-height': showNewTYPage && !isOnlyDonation}">
```

Comment:
Depending on TW compilation (sometimes TW misses classes in `:class` it seems), this could be `!tw-h-auto`. If it doesn't work, try commenting out code in `tailwind.config.js`, then uncommenting, and seeing if the class is included in the TW output.

Reply:
Yeah there's a known bug with local TW compilation, and I found that trick in a random GitHub thread recently.